examples/nextjs-shadcn/src/app/editor-example.tsx

```diff
@@ -162,11 +168,54 @@ export default function EditorExample() {
     };
   }, [checksummedAddress]);
 
+  React.useEffect(() => {
+    setInitialModActionAndContext(null);
+  }, [currentMod]);
+
   return (
     <form onSubmit={handleSubmit}>
       <div className="p-2 border border-input rounded-md">
         <EditorContent
           editor={editor}
+          onPaste={async (e) => {
```

Comment:
should be a `useCallback`, and we should probably export a helper for web for this like we do for other handlers